foundation-extensions/identity-for-edge-network/api-reference.md

```diff
@@ -141,81 +194,96 @@ Identity.getIdentities { (identityMap, error) in
   }
 }
 ```
+
+**Objective-C**
+
+**Syntax**
+
+```objectivec
++ (void) getIdentities:^(AEPIdentityMap * _Nullable map, NSError * _Nullable error)completion
+```
+
+**Examples**
+
+```objectivec
+[AEPEdgeIdentity getIdentities:^(AEPIdentityMap *map, NSError * error) {   
+    // handle the retrieved identities here    
+}];
+```
+
 {% endtab %}
 {% endtabs %}
 
-## updateIdentities
+
+## registerExtension
 
 {% tabs %}
 {% tab title="Android" %}
-### updateIdentities
-
-Update the currently known identities within the SDK. The Identity extension will merge the received identifiers with the previously saved ones in an additive manner, no identities are removed from this API.
+### registerExtension
 
-Identities with an empty _id_ or _namespace_ are not allowed and are ignored.
-
-Updating identities using a reserved namespace is not allowed using this API. The reserved namespaces are:
-
-* ECID
-* IDFA
-* GAID
+Registration occurs by passing identity for edge network extension to the MobileCore.registerExtensions API.
 
 **Java**
 
 **Syntax**
 
 ```java
-public static void updateIdentities(final IdentityMap identityMap);
+public static void registerExtension()
 ```
 
 **Example**
-
 ```java
-IdentityItem item = new IdentityItem("user@example.com");
-IdentityMap identityMap = new IdentityMap();
-identityMap.addItem(item, "Email")
-Identity.updateIdentities(identityMap);
-```
-{% endtab %}
+import com.adobe.marketing.mobile.edge.identity.Identity
 
-{% tab title="iOS — Swift" %}
-### updateIdentities
+...
+Identity.registerExtension();
+```
+{% hint style="info" %}
+If your use-case covers both Edge Network and Adobe Experience Cloud Solutions extensions, you need to register Identity for Edge Network and Identity from Mobile Core for Experience Cloud Identity Service extensions. For more details, see the [Frequently asked questions](identity-faq.md#q-i-am-using-aep-edge-and-adobe-solutions-extensions-which-identity-extension-should-i-install-and-register).
 
-Update the currently known identities within the SDK. The Identity extension will merge the received identifiers with the previously saved ones in an additive manner, no identities are removed from this API.
+{% endhint %}
+{% endtab %}
 
-Identities with an empty _id_ or _namespace_ are not allowed and are ignored.
+{% tab title="iOS (AEP 3.x)" %}
+### registerExtension
 
-Updating identities using a reserved namespace is not allowed using this API. The reserved namespaces are:
+Registration occurs by passing identity for edge network extension to the MobileCore.registerExtensions API.
```

Comment:
Same comment as above about capitalization and moving the line out of the tab group.